src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -570,9 +572,31 @@ impl Persist for VirtSandbox {
         let sandbox_state = crate::sandbox_persist::SandboxState {
             sandbox_type: VIRTCONTAINER.to_string(),
             resource: Some(self.resource_manager.save().await?),
-            hypervisor: Some(self.hypervisor.save_state().await?),
+            //hypervisor: Some(self.hypervisor.save_state().await?),
```

Comment:
This line could probably be removed, right?

Reply:
ack, will clean up once we get the correct implementation on this snippet.